docker-compose.yml

```diff
@@ -23,9 +26,8 @@ services:
   db:
     image: postgres:latest
     env_file: .env
-    ports:
-      - "5432:5432"
   redis:
     image: redis:alpine
-    ports:
-      - "6379:6379"
+    command: redis-server --save ""  
```

Comment:
abrichr 9 days ago Member
What is the reason for this change?

Reply:
https://stackoverflow.com/questions/58903982/redis-is-configured-to-save-rdb-snapshots-but-it-is-currently-not-able-to-persi

Reply:
Disabled REDIS snapshotting because was getting **MISCONF Redis is configured to save RDB snapshots, but it is currently not able to persist on disk. Commands that may modify the data set are disabled, because this instance is configured to report errors during writes if RDB snapshotting fails (stop-writes-on-bgsave-error option). Please check the Redis logs for details about the RDB error** error

Reply:
It was working before this PR. What is the reason this line is now needed?

Reply:
It would run for a while, but whenever a REDIS snapshotting event would happen, it would crash. This change was used to prevent the snapshotting event from happening, and as result, not getting this error message.